### DISCUSSION TITLE
SFlow data is not displayed.

### DISCUSSION THREAD

**Discussion by Captain-E** [akvorado/akvorado#415] [Q&A] [answered]
Hi Vincent.
First of all, i'd like to thank you for creating such an amazing flow collector.
But, unfortunately, some problems have occurred. No flows are displayed.
1. I downloaded akvorado. and i run docker containers.
2. i configured sflow on my switches. /Configuration is not wrong/
3. However no data was displayed, so i captured packets using tcpdump.

i didn't change sflow port number and enabled sflow on my aruba switch: 

However, it's not working as it should. What do you think i should do?
Any help would be appreciated. Thank you.

my akvorado.yml, docker-compose.yml file:
[docker compose.yml.txt](https://github.com/akvorado/akvorado/files/10462927/docker.compose.yml.txt)

**Comment by vincentbernat** [comment, marked as answer]
Follow the troubleshooting section of the documentation: https://demo.akvorado.net/docs/troubleshooting. Also, your capture does not show incoming sFlows and there are unanswered SNMP requests.
